tests/integration/pilot/routing_test.go

```diff
@@ -470,6 +471,7 @@ func serverFirstTestCases() []TrafficTestCase {
 		{"tcp-server", "ISTIO_MUTUAL", "STRICT", true},
 	}
 	for _, c := range configs {
+		c := c
```

Comment:
I donot think this was wrong

Reply:
c.port is captured in the close. It ways uses the last port when it shouldn't

Reply:
@hzxuzhonghu why don’t you think it is wrong?
It will work fine some times , and it will work when there is only one thing in the list, but otherwise capturing loop variable Is not right.

Reply:
IC, the closure